utils/build-script-impl

```diff
@@ -2113,7 +2113,11 @@ for host in "${ALL_HOSTS[@]}"; do
 
     for product in "${PRODUCTS[@]}"; do
         # Check if we should perform this action.
-        if ! [[ $(should_execute_action "${host}-${product}-build") ]]; then
+        tmp_product=${product}
```

Comment:
Why not simplify this further?

```sh
for product in ${PRODUCTS[@]} ; do
  product=${product//_static/}
```
should effectively give you all the changes that you need for all the products being built statically.  Generalising that means that we don't need to handle each product individually.

Reply:
At the moment the Foundation static libs dont build so I dont want `foundation_static` enabled, however once it's working I will up update to that to enable it for both. So for now just libdispatch will be enabled.